.gitignore

```diff
@@ -158,4 +158,5 @@ cython_debug/
 #.idea/
 
 # mac
-.DS_STORE
+.DS_Store
+**/.DS_Store
```

Comment:
can we avoid having two lines here? 162 is what we have for `dataprofiler` but do we have to have the `.DS_Store` too?

Reply:
actually I might be mistaken .... I think you just need line 162 here based on the git documentation [here](https://git-scm.com/docs/gitignore#:~:text=A%20leading%20%22**%22%20followed%20by%20a%20slash%20means%20match%20in%20all%20directories.%20For%20example%2C%20%22**/foo%22%20matches%20file%20or%20directory%20%22foo%22%20anywhere%2C%20the%20same%20as%20pattern%20%22foo%22.%20%22**/foo/bar%22%20matches%20file%20or%20directory%20%22bar%22%20anywhere%20that%20is%20directly%20under%20directory%20%22foo%22.)